src/video/SDL_clipboard.c

```diff
@@ -53,6 +53,44 @@ void SDL_CancelClipboardData(Uint32 sequence)
     _this->clipboard_userdata = NULL;
 }
 
+bool SDL_GetClipboardMimeTypes(char ***mime_types, size_t *num_mime_types)
```

Comment:
I'm thinking this should return `mime_types` instead of bool, for consistency with other APIs like this. They should be null-terminated (as it looks like you're doing) so num_mime_types can be a NULL pointer. You should also check to see if _this is available, e.g.
```c
    if (!_this) {
        SDL_UninitializedVideo();
        return NULL;
    }
```

Reply:
Done, changed the return type and added video init check (BTW other functions in `SDL_clipboard.c` are lacking the check)